app/controllers/sign_up/completions_controller.rb

```diff
@@ -79,16 +80,52 @@ def analytics_attributes(page_occurence)
     end
 
     def track_completion_event(last_page)
-      analytics.track_event(
-        Analytics::USER_REGISTRATION_COMPLETE,
-        analytics_attributes(last_page),
-      )
+      analytics.track_event(Analytics::USER_REGISTRATION_COMPLETE, analytics_attributes(last_page))
       GoogleAnalyticsMeasurement.new(
         category: 'registration',
         event_action: 'completion',
         method: last_page,
         client_id: ga_cookie_client_id,
       ).send_event
     end
+
+    def pii
+      @pii ||= user_session['idv']['applicant'].symbolize_keys
```

Comment:
I think this is only available immediately after proofing. If you proof, delete your session, and then sign in w/ an SP you see this:

![image](https://user-images.githubusercontent.com/963654/67691798-8fc43080-f975-11e9-89aa-7eae9759e1e7.png)

`user_session[:decrypted_pii]` is [where the PII is stored when you sign in](https://github.com/18F/identity-idp/blob/c87e29c80d8bc0118d8134c596b20a3a1213e0d4/app/services/pii/cacher.rb#L15). The old IdV flow will [also put it there after proofing](https://github.com/18F/identity-idp/blob/c87e29c80d8bc0118d8134c596b20a3a1213e0d4/app/services/idv/session.rb#L124). I think that is where we want to look for these attributes.

Reply:
It is only available after proofing.  but i call this   return pii_to_displayable_attributes if user_session['idv'].  not sure what happened here.

Reply:
The condition you are identifying here is what i fixed in this pr https://github.com/18F/identity-idp/pull/3367

Reply:
I'm not sure I see how #3367 fixes this? I can merge that PR and recreate the issue by doing this:

1. Sign up and go to `/verify`
2. Verify your identity
3. Sign out, go into chrome and delete the cookies for localhost:3000
4. Go to an example SP and sign in with IAL2
5. On the consent screen only email address is displayed

Reply:
ok i'll check it out

Reply:
that was an incredible catch.  looks like i have decrypted_pii I'll use that